libminifi/include/core/Relationship.h

```diff
@@ -78,11 +74,7 @@ class Relationship {
     return name_ < right.name_;
   }
 
-  Relationship &operator=(const Relationship &other) {
-    name_ = other.name_;
-    description_ = other.description_;
-    return *this;
-  }
+  Relationship &operator=(const Relationship &other) = default;
```

Comment:
:+1:

Not only did you simplify the code, but you also handled self-assignment of many classes, including this one. Thanks.